keyboards/handwired/aek64/readme.md

```diff
@@ -0,0 +1,17 @@
+# Alps64
+
+An Alps only 60% handwired ISO AEK based on the original AEK plate.
+
+Keyboard Maintainer: QMK Community
```

Comment:
Hi @fauxpark is this a point a mistake ? I understant that you want to add two whitespace ?

Reply:
Sorry, yes, I should have clarified - the two spaces at the end are required so that GitHub renders it properly, otherwise it will all be on one line. Newline characters don't necessarily make new lines in Markdown :)